.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,90 @@
+name: Testing Suite
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
So this means that not only will checks be run on PR creation, but also on a schedule? This is awesome.

Reply:
Big fan of it, too. Sometimes tests just break overtime from not being run.